.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -11,16 +11,16 @@ body:
     id: problem
     attributes:
       label: Issue Description
-      description: Please provide a description of the issue shown in the reproducible example.
+      description: Describe the issue.
```

Comment:
Any reason why we're shortening the description here?

If the label is already called "Issue Description", I feel a description of "Describe the issue." isn't really helpful to the user.

I kinda prefer the original description

Reply:
Just aiming for brevity. I don't think this section needs any description at all, but the `description:` attribute is mandatory so we have to put something.

I felt it was a little unclear to begin the form by referencing a later section that hasn't been introduced yet.